src/map/lua/lua_baseentity.cpp

```diff
@@ -12362,6 +12362,31 @@ sol::table CLuaBaseEntity::getNotorietyList()
     return table;
 }
 
+/************************************************************************
+ *  Function: clearEnmity()
+ *  Purpose : Clears all enmity towards target
+ *  Example : mob:clearEnmity(target)
+ *  Notes   :
+ ************************************************************************/
+void CLuaBaseEntity::clearEnmity(CLuaBaseEntity* PEntity)
+{
+    (m_PBaseEntity->objtype != TYPE_MOB);
```

Comment:
?

Reply:
From ASB
![imagen](https://github.com/LandSandBoat/server/assets/60053999/5056b52a-28f3-4c1b-a188-e69af3f5b2c2)

Reply:
XI_DEBUG_BREAK oh that whole line should be gone then eh

Reply:
Well, you probably want to check for entity type, insead of deleting the check entirely